daktari/checks/test_ssh.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from unittest.mock import patch
+
+from daktari.checks.ssh import is_ssh_configured_to_use_macos_keychain
+
+
+class TestSSH(unittest.TestCase):
+    def test_ssh_configured_correctly(self):
+        configured_ssh_config = """
+            IgnoreUnknown UseKeychain
+            UseKeychain yes
```

Comment:
Could we tweak to make them valid SSH config files (for example, in case we switch to a real SSH config parser or something later), probably just a `Host *` would do